ui-v2/tests/unit/controllers/dc-test.js

```diff
@@ -1,12 +1,12 @@
-import { moduleFor, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 
-moduleFor('controller:dc', 'Unit | Controller | dc', {
-  // Specify the other units that are required for this test.
-  // needs: ['controller:foo']
-});
+module('Unit | Controller | dc', function(hooks) {
+  setupTest(hooks);
 
-// Replace this with your real tests.
-test('it exists', function(assert) {
-  let controller = this.subject();
-  assert.ok(controller);
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let controller = this.owner.lookup('controller:dc');
+    assert.ok(controller);
+  });
 });
```

Comment:
I’d probably do it in another PR but it seems like you might as well remove this and the other placeholder controller tests! They’re pretty passé 😆